Dockerfile

```diff
@@ -15,7 +15,6 @@ COPY main.go main.go
 COPY api api/
 COPY config config/
 COPY controllers controllers/
-COPY internal internal/
```

Comment:
Shouldn't this revert commit be dropped as this is deleting new code in devel that we want?

Reply:
Right, merging main as is reverts all the 1.6.0 code from devel. We need to think harder :wink: